public/pages/createDetector/components/DataFilters/SimpleFilter.tsx

```diff
@@ -98,7 +98,7 @@ export const SimpleFilter = (props: DataFilterProps) => {
           <EuiPanel style={darkMode ? {} : lightModeStyles}>
             {values.filters.map((filter: UIFilter, index: number) => {
               return (
-                <EuiPanel className="filter-container">
+                <EuiPanel key={index} className="filter-container">
```

Comment:
Would the key of the euipanel conflict with euiaccordion since they are both using "index"?

Reply:
Good catch. React doesn't seem to throw any warnings about this, although it would probably be best to remove the 'key' prop from the `EuiAccordion`, since we only need the key to be at the top level of the component in the list. Thoughts?